sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -408,8 +408,8 @@ private[hive] class HiveClientImpl(
         lastAccessTime = h.getLastAccessTime.toLong * 1000,
         storage = CatalogStorageFormat(
           locationUri = shim.getDataLocation(h),
-          inputFormat = Option(h.getInputFormatClass).map(_.getName),
-          outputFormat = Option(h.getOutputFormatClass).map(_.getName),
+          inputFormat = Option(h.getTTable.getSd.getInputFormat),
+          outputFormat = Option(h.getTTable.getSd.getOutputFormat),
```

Comment:
When we actually read the hive table, we still use `getInputFormatClass`. So this will only affect the `DESC TABLE`, and should be OK?